CHANGELOG.md

```diff
@@ -67,6 +67,25 @@
   * Make `PayPalRequest` abstract
   * Update `PayPalRequest` setter method names and remove method chaining
   * Make `PayPalAccountBuilder` package-private
+  * Replace `CardBuilder` with `Card`
+  * Replace `BaseCardBuilder` with `BaseCard`
+  * Replace `PaymentMethodBuilder` with `PaymentMethod`
+  * Replace `UnionPayCardBuilder` with `UnionPayCard`
+  * Replace `PayPalAccountBuilder` with `PayPalAccount`
```

Comment:
It looks like this is package-private, so it doesn't need to be included in the CHANGELOG

Reply:
Yeah true, we technically may have to mark as removed since PayPalAccountBuilder was public? The CHANGELOG is gnarley though we probably want to re-visit once we go to GA to make it more concise.

Reply:
There's an entry (the last one before these changes) for making `PayPalAccountBuilder` package-private.

Also - could `VenmoAccount` be package-private too?

Reply:
Yeah we saw this too. We should definitely make `VenmoAccount` and `VisaCheckoutAccount` package-private, which leaves `Card` and `UnionPayCard` as the only merchant facing `PaymentMethod` types. It calls into question the purpose of this superclass, which seems more like a `TokenizationPayload`.

Reply:
We may address that in `beta3` if it makes sense, but _VenmoAccount_ and _VisaCheckoutAccount_ will definitely be made package-private to help reduce the surface area of the SDK.